### PR TITLE
Added global Menu element, task #33

### PR DESCRIPTION
**Pull request by vystartasv**

### CODE REVIEW

**Comment by vystartasv** [comment]
Many thanks for checking, hope I am not inventing the wheel.

File: src/frontend/App.js [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Details from './components/Details';
 import Gallery from './components/Gallery';
-import NotFound from "./components/NotFound";
-import "./App.css";
+import NotFound from './components/NotFound';
+import './App.css';
+import Menu from './components/Menu';
 
 class App extends Component {
```
Comment (rmallols) [inline src/frontend/App.js:9]: Mmmm...any reason why  you changed indentation?

File: src/frontend/components/Menu.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.menu {
+    border-radius: 100%;
+    position: fixed;
+    top: 20px;
+    left: 20px;
+    background-color: rgba(6, 6, 6, 0.6);
+    z-index: 1000;
```
Comment (rmallols) [inline src/frontend/components/Menu.css]: I wouldn't put  `100` here, feels a bit random. Why don't you set it to something like `1` or `2`?

**Review by rmallols — CHANGES_REQUESTED** [review]

File: src/frontend/App.js (hunk 1 above)
Comment (vystartasv) [reply src/frontend/App.js:9]: Configured project with eslint and applied Airbnb rules, it complained a lot more, like rect components not having jsx extensions, not using restructuring assignments, etc.

File: src/frontend/components/Menu.css (hunk 2 above)
Comment (vystartasv) [reply src/frontend/components/Menu.css]: Played around, forgot to undo, actually meant to have different value

File: src/frontend/components/Menu.css [hunk 3]
```diff
@@ -1,10 +1,10 @@
 .menu {
-    border-radius: 100%;
+    border-radius: 10%;
     position: fixed;
     top: 20px;
     left: 20px;
     background-color: rgba(6, 6, 6, 0.6);
-    z-index: 1000;
+    z-index: 50;
```
Comment (rmallols) [inline src/frontend/components/Menu.css]: Still, why 50 instead of  `1`?

**Review by rmallols — CHANGES_REQUESTED** [review]

**Review by rmallols — APPROVED** [review]
Fantastic, thanks mate :)

**Review by rmallols — APPROVED** [review]
Cool, I really like the  way you structured routing, well done Vilius :)

**Pull request merged**